astropy_helpers/extern/automodapi/automodapi.py

```diff
@@ -381,8 +383,13 @@ def process_automodapi(app, docname, source):
 
 
 def setup(app):
-    # need automodsumm for automodapi
-    app.setup_extension('astropy_helpers.sphinx.ext.automodsumm')
+
+    app.setup_extension('sphinx.ext.autosummary')
+
+    # Note: we use __name__ here instead of just writing the module name in
+    #       case this extension is bundled into another package
+    from . import automodsumm
```

Comment:
The intended behavior is that it always uses the automodsumm that lives in the same directory relative to the automodapi file, regardless of whether we are using the standalone sphinx-automodapi or (like here) bundling it in a package. The idea here is to have code that works for both the standalone and the bundled version.